ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,141 @@
+        private static void Main(string[] args)
+        {
+            var verbs = Assembly.GetExecutingAssembly().GetTypes()
+                .Where(t => t.GetCustomAttribute<VerbAttribute>() != null).ToArray();
+
+            var result = Parser.Default.ParseArguments(args, verbs)
+                .WithParsed(Run);
+        }
+
+        private static void Run(object obj)
+        {
+            MeetingService meetingService = new MeetingService();
+            RoomService roomService = new RoomService();
+
+            switch (obj)
+            {
+                case ShowMeetingsVerb showMeetingsVerb:
+                    Console.WriteLine($"Executing ShowAllMeetings in {showMeetingsVerb.Mode} mode");
+                    IEnumerable<Meeting> meetings = meetingService.GetMeetings();
+
+                    if (!meetings.Any())
+                    {
+                        Console.WriteLine("There are no meetings");
+                        break;
+                    }
+
+                    foreach (Meeting meeting in meetings)
+                    {
+                        Console.WriteLine(meeting.ToString());
+                    }
+                    break;
+
+                case AddMeetingVerb addMeetingVerb:
+                    if (addMeetingVerb.Mode == Mode.ReadAndWrite)
+                    {
+                        var createdMeeting = meetingService.AddMeeting(addMeetingVerb.Title, addMeetingVerb.Room, addMeetingVerb.MeetingStart, addMeetingVerb.Duration);
+                        Console.WriteLine("Meeting added!");
+                    }
+                    else Console.WriteLine("ReadOnly mode, change to RW to add!");
+                    break;
+
+                case AddRoomVerb addRoomVerb:
+                    if (addRoomVerb.Mode == Mode.ReadAndWrite)
+                    {
+                        var createdRoom = roomService.CreateNewRoom(addRoomVerb.Title);
+                        Console.WriteLine("Room has been created");
+                        Console.WriteLine(createdRoom);
+                    }
+                    else Console.WriteLine("ReadOnly mode, change to RW to add!");
+                    break;
+
+                case ShowRoomsVerb:
+                    var rooms = roomService.ShowRooms();
+                    if (!rooms.Any())
+                    {
+                        Console.WriteLine("No rooms yet!");
+                    }
+                    else
+                    {
+                        foreach (var room1 in rooms)
+                        {
+                            Console.WriteLine(room1.Id + " " + room1.Title + " " + room1.Meetings);
+                        }
+                    }
+                    break;
+                case ShowMeetsInRoomVerb showMeetsInRoomVerb:
+                    Room room = roomService.ShowRooms().Single(x=> x.Title == showMeetsInRoomVerb.Title);
```

Comment:
add a method to room service GetByTitle, instead of fetch all methods and filter here